store/src/main/java/com/nytimes/android/external/store3/pipeline/Mvrx.kt

```diff
@@ -0,0 +1,235 @@
+package com.nytimes.android.external.store3.pipeline
+
+import com.airbnb.mvrx.RealStateStore
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.*
+import kotlinx.coroutines.launch
+import java.util.*
+import java.util.concurrent.ConcurrentHashMap
+import kotlin.reflect.KProperty1
+
+
+internal data class MvRxTuple1<A>(val a: A)
```

Comment:
? why are we pulling classes from MvRx? or is this just tmp while working on the PR ?

Reply:
ignore just tmp